cdap-docs/_common/_themes/cdap/static/version-menu.js

```diff
@@ -35,43 +35,58 @@
   var buildURL = (function(dir){
     return versionsURL + dir + '/en/';
   });
-  var writelink = (function(dir, label){
-    document.write('<option value="' + buildURL(dir) + '">Version ' + label + '</option>');
+  var writeLink = (function(dir, label){
+    document.write('<option value="' + buildURL(dir) + '">' + label + '</option>');
+  });
+  var writeVersionLink = (function(dir, label){
+    writeLink(dir, 'Version ' + label);
   });
   window.versionscallback = (function(data){
+    var ess;
     if (data) {
       document.write('<li class="versions">');
       document.write('<select id="' + versionID + '" onmousedown="window.currentversion=this.value;" onchange="window.gotoVersion(\'' + versionID + '\')">');
     }
-    var ess = "s";
-    if (data.development && data.development.length > 0) {
-      if (data.development.length == 1) {
-        ess = "";
-      }
+    
+    if (data.development && data.development.length > 0 && data.development[0]) {
```

Comment:
If we don't support IE8 then we could simplify the `if` condition to something like this,

```if (Array.isArray(data.development) && data.development.length) { ... }```